proposals/NNNN-DXIL-Data-Scalarization.md

```diff
@@ -0,0 +1,62 @@
+# DXIL Data Scalarization
+
+* Proposal: [NNNN](NNNN-DXIL-Data-Scalarization.md)
+* Author(s): [Farzon Lotfi](https://github.com/farzonl)
+* Sponsor: [Farzon Lotfi](https://github.com/farzonl)
+* Status: **Under Consideration**
+
+## Introduction
+
+In Proposal [0009](0009-DXIL-Function-Scalarization.md) we covered scalarization
+of call instructions, and vector operations like math ops, logical ops,
+bitcasts, loads, and stores. The goal of this proposal is to present a solution
+for scalarizing data structures via data layout transformations. As a note,
+any case that results in an implicit cbuffer will not be covered by this
+proposal.
+
+## Motivation
+
+As mentioned in [DXIL.rst](https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#vectors)
```

Comment:
If you use an out-of-line link here the paragraph will flow in the text view a little better:
```
As mentioned in [DXIL.rst] etc etc ...

[DXIL.rst]: https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#vectors
```